Decred Wallet/Features/TransactionDetails/TransactionDetailsViewController.swift

```diff
@@ -8,262 +8,294 @@ import UIKit
+            self.txIconImageView.image = UIImage(named: "ic_fee")
+            self.txAmountLabel.attributedText = amountString
+        }
     }
-    
-    @objc func showOptionsMenu(sender: UIBarButtonItem) {
-        let alertController = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
-        
-        let cancelAction = UIAlertAction(title: LocalizedStrings.cancel, style: .cancel, handler: nil)
-        
-        let copyTxHash = UIAlertAction(title: LocalizedStrings.copyTransactionHash, style: .default, handler: { (alert: UIAlertAction!) -> Void in
-            self.copyText(self.transaction.hash)
-        })
-        
-        let copyRawTx = UIAlertAction(title: LocalizedStrings.copyRawTransaction, style: .default, handler: { (alert: UIAlertAction!) -> Void in
-            self.copyText(self.transaction.hex)
-        })
-        
-        let viewOnDcrdata = UIAlertAction(title: LocalizedStrings.viewOnDcrdata, style: .default, handler: { (alert: UIAlertAction!) -> Void in
-             if BuildConfig.IsTestNet {
-                self.openLink(urlString: "https://testnet.dcrdata.org/tx/\(self.transaction.hash)")
-             } else {
-                self.openLink(urlString: "https://dcrdata.decred.org/tx/\(self.transaction.hash)")
-            }
-        })
-        
-        alertController.addAction(cancelAction)
-        alertController.addAction(copyTxHash)
-        alertController.addAction(copyRawTx)
-        alertController.addAction(viewOnDcrdata)
-        
-        if let popoverPresentationController = alertController.popoverPresentationController {
-            popoverPresentationController.barButtonItem = self.navigationItem.rightBarButtonItems![0]
+
+    func displayTicketPurchaseInfo(_ transaction: Transaction) {
+        self.titleLabel.text = " \(LocalizedStrings.voted)"
+        self.txIconImageView.image =  UIImage(named: "ic_ticketVoted")
+
+        let lastBlockValid = TransactionDetails(
+            title: LocalizedStrings.lastBlockValid,
+            value: NSMutableAttributedString(string: String(describing: self.transaction.lastBlockValid)),
+            isCopyEnabled: false
+        )
+        generalTxDetails.append(lastBlockValid)
+
+        let voteVersion = TransactionDetails(
+            title: LocalizedStrings.version,
+            value: NSAttributedString(string: "\(self.transaction.voteVersion)"),
+            isCopyEnabled: false
+        )
+        generalTxDetails.append(voteVersion)
+
+        let voteBits = TransactionDetails(
+            title: LocalizedStrings.voteBits,
+            value: NSAttributedString(string: self.transaction.voteBits),
+            isCopyEnabled: false
+        )
+        generalTxDetails.append(voteBits)
+        self.txAmountLabel.attributedText = Utils.getAttributedString(
+            str: "\(self.transaction.dcrAmount.round(8))",
+            siz: 20,
+            TexthexColor: UIColor.appColors.darkBlue
+        )
```

Comment:
Append these in the recommended `prepareTransactionDetails` function that is called on `viewDidLoad`.